overlays/holo-nixpkgs/hpos-reset/hpos-reset.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Simple shell script for resetting a HoloPort to default settings and clearing user
+##
+
```

Comment:
It would be good to configure the shell in the standard way for scripts.
```suggestion
set -ex
```

`set -e` means that the script will exit if any of the commands errors, rather than proceeding as normal. `set -x` means that the script will print out each command it's running as it runs them

Reply:
I removed `set -e` because otherwise the script fails at `rm -rf /var` because for some reason it throws some weird error about being unable to delete `/var/empty`

Reply:
In that case you can add `|| echo "ignoring failure from rm command"` after the `rm` command in specific